src/components/form/ScopesForm/ScopeInput.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ScopeInput = ({
```

Comment:
I do not think there is a need for such "découpage" ;)

To my opinion, this component does nothing but grouping input with label. And it introduce to much props forwarding (about 20 lines) which can be painful to maintain.